client/gui/labeler.ts

```diff
@@ -1,9 +1,25 @@
-"use strict";
+import * as _ from "underscore";
+import * as $ from "jquery";
 
-const _ = require("underscore");
-const $ = require("jquery");
+import {KEYS} from "./keyboard";
+import type {GUI} from ".";
 
-class Labeler {
+function flashDropdown(name: string, inputName?: string) {
```

Comment:
This function got resurrected from an old commit where it was removed -- we need _something_ like it because we call a function by this name a few times below... at the moment, though, the `Labeler` functionality is disabled (hidden) altogether (see e951f7255e5b02b8c545af13a4a4e1e53f67054f).